nix/module-options.nix

```diff
@@ -174,6 +174,19 @@
         type = listOf str;
         default = [];
       };
+      executor = mkOption {
+        description = "Select which executor will be used for evaluating and building the NixOS system configuration. Specify settings used by that executor.";
+        type = lib.types.submodule {
+          freeformType = (pkgs.formats.yaml { }).type;
```

Comment:
Since we currently only have the nix executor, this configuration option is not required.
Is it an issue for you to remove it for now?

Also, i would prefer to avoid `freeformType` when possible.